test/server/cards/03_TWI/units/AT-TEVanguard.spec.ts

```diff
@@ -0,0 +1,33 @@
+describe('AT-TE Vanguard', function() {
+    integration(function(contextRef) {
+        describe('AT-TE Vanguard\'s ability', function() {
+            beforeEach(function () {
+                contextRef.setupTest({
+                    phase: 'action',
+                    player1: {
+                        hand: ['power-of-the-dark-side']
+
+                    },
+                    player2: {
+                        groundArena: ['atte-vanguard']
+                    }
+                });
+            });
+
+            it('should create 2 Clone Tropper tokens when defeated', function () {
+                const { context } = contextRef;
+
+                context.player1.clickCard(context.powerOfTheDarkSide);
+                expect(context.player2).toBeAbleToSelectExactly([context.atteVanguard]);
```

Comment:
Similar comment as Battle Droid Legion - we're not testing PotDS so we don't need expectations around it

Reply:
fixed up